adapters/repos/db/inverted/objects.go

```diff
@@ -344,6 +346,7 @@ func stringsFromValues(prop *models.Property, values []interface{}) ([]string, e
 func (a *Analyzer) analyzePrimitiveProp(prop *models.Property, value interface{}) (*Property, error) {
 	var hasFrequency bool
 	var items []Countable
+	PropertyLength := -1 // will be overwritten for string/text, signals not to add the other types.
```

Comment:
why this variable is a public one?

Reply:
no reason, it is just a local variable so it shouldn't have any effect anyways? But I renamed it to lowercase